src/lib/common/topology_sampler.h

```diff
@@ -7,4 +7,7 @@
 
 #include <stdint.h>
 
+#define TOPOLOGY_SAMPLER_MAX_NODE_LIST_SIZE (24)
```

Comment:
Fairly arbitrary! Don't think we'll be hitting this any time soon!

Reply:
Another source for a fairly arbitrary number — we're defining this field in the satellite message to the Sofar backend as a 4-bit integer representing 1-16 where the bridge is always present and there may be up to 15 more.